### ISSUE TITLE
Clearly state that this project is archived/ deprecated

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/polymer-app-template#14]
[This suggests](https://github.com/eXist-db/usermanager/pull/21#issuecomment-477967053) that the template is deprecated. we should add a note to the readme and archive this repo if this is the case

**Comment by JoernT** [comment]
yes, this repo should be marked as deprecated as it relies on an old version of polymer and tooling.
